### DISCUSSION TITLE
Is the new Layouts RFC ready to try?

### DISCUSSION THREAD

**Discussion by yousefelgoharyx** [vercel/next.js#40427] [Help] [answered]
Is there a way to enable working with the new layout system? I installed the latest nextjs version but it does not seems to see the app directory

**Comment by leerob** [comment]
Hey! The new layouts work has not been released yet. There will be another release with more details on how to try it out 😄 We're working on it as fast as we can.

**Reply by yousefelgoharyx** [reply]
Oh okay thanks for explaining

**Comment by IRediTOTO** [comment]
And you wrote a blog about it. I was too excited to try it, but it did not work. Should have **Red message** about not ready to use :|

**Comment by leerob** [comment, marked as answer]
**Update:** We’re excited to announce [Next.js 13](https://nextjs.org/13), which includes the new `app/` directory (beta) with support for nested layouts, colocation of data fetching, streaming, Server Components by default, and much more.

Inside the `app/` directory, there is a powerful new way to fetch data with React’s `use()` hook and the extended Web `fetch()` API. This allows you to colocate data fetching directly inside components for the most flexibility, including inside layouts.

You can leave feedback here: https://github.com/vercel/next.js/discussions/41745
